Modules/ITS/include/ITS/ITSTrackTask.h

```diff
@@ -118,6 +125,149 @@ class ITSTrackTask : public TaskInterface
+    double thetaR;
+    double Bz;
+    double sigma_meas[2][NLayer] = {{45,45,45,55,55,55,55},
+                                    {40,40,40,40,40,40,40}}; //um unit
+    double sigma_msc[2][NLayer]  = {{30,30,30,110,110,110,110},
+                                    {25,25,25,75,75,75,75}}; //um unit
+
+    se_circlefitXY() = default;
+    se_circlefitXY(std::vector<TVector3> &h, double tR, double bz) : fHits(h), thetaR(tR), Bz(bz) { };
+
+    void loadparameters(double arrpar_meas[][NLayer], double arrpar_msc[][NLayer]){
+      for(int a = 0; a < 2; a++){
+        for(int l = 0; l < NLayer; l++){
+          sigma_meas[a][l] = arrpar_meas[a][l];
+          sigma_msc[a][l] = arrpar_msc[a][l];
+        }
+      }
+    };
+
+    void init(){
+      fHits.clear();
+      thetaR = 0;
+      Bz = 0;
+    };
+
+    void set(std::vector<TVector3> &h, double tR, double bz){
+      fHits = h;
+      thetaR = tR;
+      Bz = bz;
+    };
+
+    double getsigma(double R, int L, double B, int axis){
+      //R : cm
+      //B : T
+      if(L<0 || L>=NLayer) return 1;
+      double aL   = sigma_meas[axis][L]*1e-4; //um -> cm
+      double bL   = sigma_msc[axis][L]*1e-4;  //um -> cm
+      double Beff = 0.3*B;
+      double sigma = std::sqrt( std::pow(aL,2) + std::pow(bL,2)/(std::pow(Beff,2)*std::pow(R*1e-2,2)));
+
+      return sigma;
+    };
+
+    // calculate distance line-point
+    double distance2(double x, double y, const double *p, double tR, int charge) {
+
+      double R = std::abs(1/p[0]);
+    
+      double Xc  = p[0] > 0 ? R*std::cos(p[1] + tR + 0.5*TMath::Pi()) : R*std::cos(p[1] + tR - 0.5*TMath::Pi());
+      double Yc  = p[0] > 0 ? R*std::sin(p[1] + tR + 0.5*TMath::Pi()) : R*std::sin(p[1] + tR - 0.5*TMath::Pi());   
+    
+      double dx = x - (Xc + p[2]);
+      double dy = y - (Yc + p[3]);    
+      double dxy = R - std::sqrt(dx*dx + dy*dy);
+    
+      double d2 = dxy*dxy;
+      return d2;
+    };
+
+    // implementation of the function to be minimized
+    double operator() (const double *par) { //const double -> double
+      assert(fHits != 0);    
+
+      int nhits       = fHits.size();
+      double sum      = 0;
+
+      double charge    = par[0]>0 ? +1 : -1;
+      double RecRadius = std::abs(1/par[0]);
+
+      double Sigma_tot[NLayer];
+      double sum_Sigma_tot = 0;
+      for(int l = 0; l < nhits; l++){
+        Sigma_tot[l] = getsigma(RecRadius, fHits[l].Z(), Bz, 1);
+        sum_Sigma_tot += 1/(std::pow(Sigma_tot[l],2));
+      } 
+    
+      for (int l = 0; l < nhits; l++) {
+         double d = distance2(fHits[l].X(), fHits[l].Y(), par, thetaR, charge)/(std::pow(Sigma_tot[l],2));  
+         sum += d;
+      }
+      sum = sum / sum_Sigma_tot;
+
+      return sum;
+    };
+
+  };
+
+  se_circlefitXY fitfuncXY;
+
+  void lineFitDZ(double* zIn, double* betaIn, double* parz, double Radius, bool vertex, std::vector<bool> hitUpdate);
+
+  double sigma_meas[2][NLayer] = {{45,45,45,55,55,55,55},
```

Comment:
Is it a duplicate on of line 165?

Reply:
No, one is data member of ITSTrackTask, and another one is data member of struct "se_circlefitXY".

I renamed the former case as "mSigmaMeas, mSigmaMsc" to have similar patterns for the normal data member in ITSTrackTask.